src/vcnl4040.toit

```diff
@@ -0,0 +1,483 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
+// See the LICENSE file in the package directory.
```

Comment:
Hmm. Should we write that in our guidelines?
I mean, that package files should have that line.
is it necessary?

Reply:
Here is what Dart does.  This is after the change. The old way was that every single file had to contain the entire license:

// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.